assignment/week_3/RecommendedMenu/src/components/RecommendMenu.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect } from "react";
+import TasteRecommendation from "./TasteRecommendation";
+import CountDown from "./CountDown";
+
+const RecommendMenu = (props) => {
+  const category = props.category;
+  const step = props.step;
```

Comment:
아 막 하나하나 다 받아오는걸 작성해줄 필요없이 props만 써주면 되네..?

Reply:
타스에서는 오빠처럼 하나씩 props로 전달해줘야 하는데, 자스는 그냥 props로만 넘겨줘도 되더라고 ?!
근데 내가 둘 다 써보니까 자스는 props로만 넘겨줘도 돼서 편하긴 한데, 
또 props의 수가 많아지면서 어떤걸 전달해주고 전달받았는지, 각 props의 타입이 뭔지 등등 헷갈리는게 많이 생기더라 ..!!
어차피 나중에 타스를 쓰게 될거라면 각각의 props를 넘겨주는 방식도 쓰게 될테니까 두가지 방식 모두 알고 있으면 좋을듯 !

Reply:
props로 받아온 객체에 구조분해할당으로 접근해서 받아오려고 중괄호로 감싸서 하나씩 가져오는 원리입니당 ~!

```suggestion
const RecommendMenu = ({category, step}) => {
 // 여기서 category와 step에 재할당해줄 필요 없이 그냥 category, step으로 쓰면 됨!
```